### DISCUSSION TITLE
Press 'e' to edit / 'm' to move the Issue directly from a list

### DISCUSSION THREAD

**Discussion by o4rz3l** [ankitpokhrel/jira-cli#403] [Ideas]
Hi, 
i'd like to be able to press the 'e' key on a list of issues to directly edit it or to press 'm' to move it!
Thanks for the great tool!
Best regards, 
o4rz3l

**Comment by o4rz3l** [comment]
BTW 'c' for comment would be nice as well ;)

**Comment by JacobCallahan** [comment]
I've been watching this for a while and would love to see it implemented!
However, I wonder what that implementation would look like.

Here are a few unsolicited ideas:
1. Edit views - Similar to current views, but can ?tab? through fields to edit them. Submits update(s) upon view ?exit?.
2. Prepared follow-up command - When an action is desired (comment/edit) add prepped follow-up command to the user's clipboard and exit. They can then paste the command and tailor it before submitting it.
3. Jump-to-action - like the previous, but skip to the state a user would be at when submitting a command that offers interaction (add a comment, move state)

Edit views are likely the most difficult to implement, but they would be pretty nice!

**Comment by harkairt** [comment]
I'd really love to move from the shitty web interface to this TUI, but this basic workflow feature is missing for me.

**Comment by ankitpokhrel** [comment]
Hey folks, thank you for the input. Option to move issue directly from the TUI is mostly done with #544. It would be great if someone could help to test this out.

To install, you can use one of the following methods.

```sh
# Docker
$ docker run -it --rm ghcr.io/ankitpokhrel/jira-cli:feat-move-in-tui

# Golang
$ go install github.com/ankitpokhrel/jira-cli/cmd/jira@feat-move-in-tui

# Compile manually
# See https://github.com/ankitpokhrel/jira-cli#development
```

To test:
- Navigate to the issue
- Press `m`
- Navigate between options using `TAB` or left and right arrow keys
- Press ENTER to select new status

![move-tui](https://user-images.githubusercontent.com/2364546/210221930-05fd24df-c104-4934-989c-662b992d860b.gif)

**Reply by o4rz3l** [reply]
Hi @ankitpokhrel, thanks for the update. I installed it an it works great so far. What i noticed is, that there is a set resoultion screen in front of e.g. Done, then this is not considered (i think this is a general problem). The problem with this is, that the CLI moves the issue to Done but it is unresoveled, which is not so good.
Is there something i can do?

Thanks for your great work!

**Reply by ankitpokhrel** [reply]
Thanks for checking @o4rz3l

Setting resolution and assignee when moving the issue depends on the workflow and is not enabled by default. These will be added in future iterations, target for now is to ship the basic feature.

**Comment by mmlb** [comment]
@ankitpokhrel any chance `e` for edit makes sense too?

**Comment by T4cC0re** [comment]
I would love to be able to at least watch or unwatch an issue. While full `e` capabilities would be nice, maybe that is a good first iteration?
